content/hi/blue_green_deployment.md

```diff
@@ -0,0 +1,15 @@
+---
+title: ब्लू ग्रीन डिप्लॉयमेंट (Blue Green Deployment)
+status: Completed
+category: संकल्पना
+---
+
+## यह क्या है
+ब्लू-ग्रीन डिप्लॉयमेंट (Blue-green deployment) एक रणनीति है जिससे चलते हुए कंप्यूटर सिस्टम्स को काम से काम डाउनटाइम के डोरान अपडेट किया जा सकता है। ऑपरेटर दो एनवीरोंमेंट्स(environments) बनाए रखता है, जिन्हे "ब्लू" ("blue") और "ग्रीन" ("green") कहा जाता है। एक उत्पादन ट्रैफ़िक प्रदान करता है (वह संस्करण जो सभी उपयोगकर्ता वर्तमान में उपयोग कर रहे हैं), जबकि दूसरा अपडेट किया जाता है। एक बार टेस्टिंग (ग्रीन) वातावरण पर परीक्षण समाप्त हो जाने के बाद, उत्पादन ट्रैफ़िक को स्विच किया जाता है (अक्सर लोड बालसर के उपयोग के माध्यम से)। ध्यान दें कि ब्लू-ग्रीन डिप्लॉयमेंट का मतलब आमतौर पर पूरे एनवीरोंमेंट्स को बदलना है, जिसमें कई  सर्विसेज (services) शामिल हैं, सभी एक साथ। भ्रामक रूप से, कभी-कभी इस शब्द का प्रयोग सिस्टम के भीतर व्यक्तिगत सर्विसेज के संबंध में किया जाता है। इस अस्पष्टता से बचने के लिए, अलग-अलग कंपोनेंट्स का जिक्र करते समय "जीरो-डाउनटाइम डिप्लॉयमेंट" ("zero-downtime deployment") शब्द को प्राथमिकता दी जाती है।
```

Comment:
line 5: `बलैंसर ` instead of `बालसर`
line 7 : there's a word for services in hindi already maybe we could use `सेवाएं`